public/doc/openapi-3-v1.1.json

```diff
@@ -2927,7 +2927,7 @@
           }
         }
       },
-      "PortfolioItem": {
+      "ShowPortfolioItem": {
```

Comment:
@eclarizio Isn't this a new object definition? Does this mean any time we add a new attribute or change the behavior we will add a new object definition. The metadata was meant to hide all of those and allow us to add new fields without having to add new object definitions

Reply:
@mkanoor For a specific endpoint, yes. But again, this is just a documentation, if you add more data to the metadata in the model, it would still show up on the index call despite not changing anything in the openapi json. I only added this here as an indication that something is different with the show endpoint. This whole PR would still work and return the additional "orderable" metadata even if I didn't change the openapi json.